app/experimenter/nimbus-ui/src/components/TableSummary/index.tsx

```diff
@@ -56,6 +58,43 @@ const TableSummary = ({ experiment }: TableSummaryProps) => {
             )}
           </td>
         </tr>
+        <tr>
+          <th>Risk mitigation checklist</th>
+          <td data-testid="experiment-risk-mitigation-link">
+            {experiment.riskMitigationLink ? (
+              <LinkExternal href={experiment.riskMitigationLink}>
+                <span className="mr-1 align-middle">
+                  {experiment.riskMitigationLink}
+                </span>
+                <ExternalIcon />
+              </LinkExternal>
+            ) : (
+              <NotSet />
+            )}
+          </td>
+        </tr>
+        {experiment.documentationLinks &&
+          experiment.documentationLinks?.length > 0 && (
+            <tr>
+              <th>Additional links</th>
+              <td data-testid="experiment-additional-links">
+                {experiment.documentationLinks.map((documentationLink, idx) => (
+                  <React.Fragment key={`doc-link-${idx}`}>
+                    <LinkExternal
+                      href={documentationLink.link}
+                      data-testid="experiment-additional-link"
+                    >
+                      <span className="mr-1 align-middle">
+                        {documentationLink.title}
+                      </span>
+                      <ExternalIcon />
+                    </LinkExternal>
+                    {idx !== experiment.documentationLinks!.length - 1 && ", "}
```

Comment:
Re: 
![](https://user-images.githubusercontent.com/6392049/104638253-ef3e5700-5673-11eb-9b12-8efa4362cbed.png)

Would it make more sense to display each link on a separate line to give them a bit more space? (basically replacing the comma here with a `<br/>`)

Reply:
I tend to agree. @AnaMedinac thoughts?

Reply:
Update she's in!